keip-default-image/pom.xml

```diff
@@ -58,6 +58,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
```

Comment:
Same issue with duplicate dependency here

Reply:
Ah, I think these came back in the rebase